### DISCUSSION TITLE
Sort apps into categories

### DISCUSSION THREAD

**Discussion by terjene** [LawnchairLauncher/lawnchair#3374] [Ideas]
I like to sort apps into folders. When categorizing apps it would be nice to only see apps that are not already categorized into another category.
First i put all my car-apps into a folder called cars. Then when selecting apps for my bank-folder, I still see my car-apps. They are already categorized, and when I have a lot of apps, this is a pain..

**Comment by ghost** [comment]
I brought it up in the telegram group a long time ago and someone responded back to me that they are planning on implementing features they had in the prior Lawnchair version into this, but it'll take time since they are building the whole thing from scratch.
